pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicyrule/computefirewallpolicyrule-egress-full-direct/_http.log

```diff
@@ -803,6 +803,7 @@ X-Goog-Request-Params: firewall_policy=${firewallPolicyID}
 {
   "action": "allow",
   "direction": "EGRESS",
+  "kind": "compute#firewallPolicyRule",
```

Comment:
This change confirms that `kind` is now being sent in the PATCH request body. As noted in the controller comment, this is a side effect of the fix. Since it matches real GCP behavior (or is at least accepted), it is acceptable.